flexmeasures/api/v1_1/implementations.py

```diff
@@ -177,6 +180,9 @@ def post_weather_data_response(  # noqa: C901
             weather_sensor = get_weather_sensor_by(
                 weather_sensor_type_name, latitude, longitude
             )
+            if type(weather_sensor) == ResponseTuple:
```

Comment:
The way `get_weather_sensor_by` returns a WeatherSensor or a ResponseTuple isn't particularly beautiful. I consider this just a small fix in an older API version. In v2, we'd use Marshmallow for such validation.